gnc/sub8_perception/nodes/follow_orange_pipes.py

```diff
@@ -54,6 +54,11 @@ def get_ncc_scale(self):
 
         a = np.array(self.occ_grid.cam.project3dToPixel([0.0, self.channel_width, height]))
         b = self.occ_grid.cam.project3dToPixel([0.0, 0.0, height])
+        try:
+            int(np.linalg.norm(a - b) / 2.0)
+        except:
```

Comment:
Could have been something else was not initialized also, I will remove the try and let you know what error it gives if it ever happens